Dalamud/Interface/Internal/Notifications/NotificationManager.cs

```diff
@@ -203,20 +233,40 @@ internal enum Phase
             Expired,
         }
 
+        /// <summary>
+        /// Gets the notification ID.
+        /// </summary>
+        internal int NotificationId { get; } = notificationIdCounter++;
+
         /// <summary>
         /// Gets the type of the notification.
         /// </summary>
         internal NotificationType NotificationType { get; init; }
 
         /// <summary>
-        /// Gets the title of the notification.
+        /// Gets or sets a value indicating whether to force the use of monospace font.
+        /// </summary>
+        internal bool UseMonospaceFont { get; set; }
+
+        /// <summary>
+        /// Gets the action buttons to attach to this notification.
+        /// </summary>
+        internal List<(string Text, Action ClickCallback)> Actions { get; } = new();
```

Comment:
People have been asking for this, so we should remember to make it public at some point

Reply:
Merge this for now if everything else looks alright; I'll make another PR that deals with interactable notifications better later